libs/wire-subsystems/test/unit/Wire/MiniBackend.hs

```diff
@@ -352,9 +370,29 @@ interpretMaybeFederationStackState maybeFederationAPIAccess localBackend teamMem
     . inMemoryUserStoreInterpreter
     . liftBlockListStoreState
     . inMemoryBlockListStoreInterpreter
+    . liftActivationCodeStoreState
+    . inMemoryActivationCodeStoreInterpreter
+    . liftInvitationInfoStoreState
+    . liftInvitationCodeStoreState
+    . inMemoryInvitationCodeStoreInterpreter
     . miniGalleyAPIAccess teamMember galleyConfigs
     . runUserSubsystem cfg
 
+liftInvitationInfoStoreState :: (Member (State MiniBackend) r) => Sem (State (Map InvitationCode StoredInvitationInfo) : r) a -> Sem r a
+liftInvitationInfoStoreState = interpret \case
+  Polysemy.State.Get -> gets (.invitationInfos)
+  Put newAcs -> modify $ \b -> b {invitationInfos = newAcs}
+
+liftActivationCodeStoreState :: (Member (State MiniBackend) r) => Sem (State (Map EmailKey (Maybe UserId, ActivationCode)) : r) a -> Sem r a
+liftActivationCodeStoreState = interpret \case
+  Polysemy.State.Get -> gets (.activationCodes)
+  Put newAcs -> modify $ \b -> b {activationCodes = newAcs}
+
+liftInvitationCodeStoreState :: (Member (State MiniBackend) r) => Sem (State (Map (TeamId, InvitationId) StoredInvitation) : r) a -> Sem r a
+liftInvitationCodeStoreState = interpret \case
+  Polysemy.State.Get -> gets (.invitations)
+  Put newInvs -> modify $ \b -> b {invitations = newInvs}
```

Comment:
same for this - I think this should all be replaced by handing something like a `Lens'` to a more generic function - but maybe something to think about in the future ;)

Reply:
Yes but out of scope.